tests/integration/components/notification-card-test.js

```diff
@@ -0,0 +1,83 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'website-www/tests/helpers';
+import { render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+module('Integration | Component | notification-card', function (hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders with title and description', async function (assert) {
+    assert.expect(3);
+
+    this.set('title', 'Test Title');
+    this.set('description', 'Test Description');
+
+    await render(hbs`
+      <NotificationCard
+        @title={{this.title}}
+        @description={{this.description}}
+      />
+    `);
+
+    assert
+      .dom('[data-test-notification-card]')
+      .exists('Component wrapper exists');
```

Comment:
We should not be testing for this. This wrapper is of no significance to the user. Tomorrow, maybe we want to refactor using two wrappers, or maybe the wrapper comes from a separate component.